### ISSUE TITLE
Document Collection API on the endpoints page

### ISSUE THREAD

**Issue by Wauplin** [huggingface/hub-docs#967]
Follow-up of https://github.com/huggingface/huggingface_hub/issues/1682#issuecomment-1728161950 and to be done once https://github.com/huggingface/hub-docs/pull/962 is merged cc @julien-c 
Opening a separate issue as it's unrelated to the huggingface_hub implementation per-se.

I can take care of it.

**Comment by Wauplin** [comment]
Closed by https://github.com/huggingface/hub-docs/pull/973.
